modules/zip_importer/includes/importer.inc

```diff
@@ -421,6 +441,17 @@ EOXML;
   }
 
   /**
+  *  This function is the php magic function to reconstruct the file paths of xsl which were stored in the 
+  *  static variables. This simply calls initialize_paths method once more to initialize the paths.
+  *  
+  *  This method will be called when we unserialize this object after fetching it from the database.
```

Comment:
Comments shouldn't try to describe core PHP functionality in detail, nor describe exactly what will happen inside the function... It should suffice to say something like: "Implements the PHP magic method __wakeup().".